SR2024/2024-03-13-competition-rsvp-chase.md

```diff
@@ -0,0 +1,21 @@
+---
+to: SR2024 teams who still haven't RSVP'd
+subject: SR2024 RSVP & check-in
+---
+
+Hi $TODO_NAME,
+
+{# either #}
+It was great to see your team at the Virtual Competition this weekend.
+{# or #}
+We missed you at the Virtual Competition this weekend.
+{# / #}
```

Comment:
Nit: We've historically used jinja-style syntax for these. This also makes it more explicit what each case is conditional on.

```suggestion
{% if attended_virtual_comp %}
It was great to see your team at the Virtual Competition this weekend.
{% else %}
We missed you at the Virtual Competition this weekend.
{% endif %}
```